### PR TITLE
feat/69/mobile terms and con

### PR DESCRIPTION
**Pull request by RainbowGiantSquid**
- feat: add privacy_policy and terms_and_conditions skeleton

Testing:
1. navigate to /mobile/terms_and_conditions
2. in the browser inspector select different mobile devices to check responsiveness
3. Designs in screenshot - there are responsive margins and fonts so the exact measures may vary based on device

Designs - https://unep-wcmc.codebasehq.com/projects/bits-and-bugs/tickets/68

Ticket - https://unep-wcmc.codebasehq.com/projects/bits-and-bugs/tickets/69

image

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -12,27 +12,34 @@
...
   <![endif]-->
 
-  <%= stylesheet_link_tag    "admin" %>
+  <%= stylesheet_link_tag "admin" %>
+  <%= stylesheet_link_tag "mobile" %>
   <%= javascript_include_tag "application" %>
   <%= csrf_meta_tags %>
 </head>
 <body>
+<% if controller.controller_name != 'mobile' %>
```
Comment (sergiomarrocoli) [inline app/views/layouts/application.html.erb:21]: indentation

**Review by sergiomarrocoli — CHANGES_REQUESTED** [review]
one comment about moving to a separate layout, but otherwise everything looks good

**Review by sergiomarrocoli — CHANGES_REQUESTED** [review]
is the close button supposed to do anything? redirect to home maybe?

**Comment by RainbowGiantSquid** [comment]
> 

Nothing for now. This is gonna be implemented far in the future.

**Review by sergiomarrocoli — APPROVED** [review]

**Pull request merged**